src/apps/backend/modules/comment/comment_service.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+from modules.comment.errors import CommentCreationError, CommentNotFoundError, CommentServiceError, CommentUpdateError
+from modules.comment.internal.comment_reader import CommentReader
+from modules.comment.internal.comment_writer import CommentWriter
+from modules.comment.types import Comment, CreateCommentParams
+
+
+class CommentService:
+    @staticmethod
+    def create_comment(*, params: CreateCommentParams) -> Comment:
+        try:
```

Comment:
We should avoid making service layer contain this logic and should be in the comment writer itself.